### ISSUE TITLE
Feature Request: Add Git-Flow like SourceTree

### ISSUE THREAD

**Issue by Tekkharibo** [gitahead/gitahead#348]
Add the Git Flow actions like SourceTree

**Comment by Murmele** [comment]
Can you show a screenshot? Which actions?

**Comment by oligu** [comment]
This is now from SourceTree, but from the [VS Code GitFlow extension](marketplace.visualstudio.com/items?itemName=vector-of-bool.gitflow). The actions are:
- Initialize (define the names of the different branches)
- Feature: start
- Feature: finish
- Feature: rebase
- Hotfix: start
- Hotfix: finish
- Release: start
- Release: finish
- Bugfix: start
- Bugfix: finish
- Bugfix: rebase

**Comment by Nebula83** [comment]
Any updates on this feature?

**Comment by timvankatwijk** [comment]
Yes! This would be a magnificent feature to have. I'd like to add to the list that @oligu provided the oft-forgotten action:

- Sustain: start

**Comment by ricardoper** [comment]
+1 - This feature it's essential to many teams.

**Comment by Bit-Line** [comment]
+1 I would like to see that feature soon too, as it stands for most common workflow standards.

**Comment by rzamoramx** [comment]
+1 this feature is essential for any dev

**Comment by btey** [comment]
+1
